ManPhamQuang/COSC2769-rmit
# Feat/expert dashboard

**Changelog**

- added `/dashboard` and `/dashboard/profile` pages
- added `useBreakpoint` hook for detecting screen sizes
- added a new tailwind custom breakpoint: `latop:` which has a `min-width: 900px`

**Known-issues**

- `/dashboard` routes are not protected (needs to redirect unlogged-in or non-experts users)
- Rooms Table is view-only at the moment
- `Profile` component is currently a static form
- `Log Out` button's function is not implemented

**How to test**

- Check out this PR
- Login (make sure to create an account first if you haven't already)
- Create some rooms (if your account already have some rooms, skip this step)
- Navigate to `/dashboard` and `/dashboard/profile` to test all acceptance criteria in #36
- Use slow 3G or fast 3G to test Rooms Table loading skeleton
